packages/components/src/checkbox-control/style.scss

```diff
@@ -50,9 +64,10 @@
 .components-checkbox-control__input-container {
 	position: relative;
 	display: inline-block;
-	margin-right: 12px;
+	margin-right: $grid-unit-15;
 	vertical-align: middle;
 	width: var(--checkbox-input-size);
+	height: var(--checkbox-input-size);
 	aspect-ratio: 1;
```

Comment:
Was this `height` necessary? It makes the `aspect-ratio` redundant.

I think we also need a `flex-shrink: 0` here to prevent squishing:

<img width="298" alt="Element being squished" src="https://github.com/WordPress/gutenberg/assets/555336/1ec1d28f-5e74-4895-b711-20ff3aa82114">

Reply:
It was, but moving to a HStack seems to have made it redundant :)